Mathlib/Combinatorics/Additive/Corner/Defs.lean

```diff
@@ -0,0 +1,103 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Combinatorics.Additive.FreimanHom
+
+/-!
+# Corners
+
+This file defines corners, namely triples of the form `(x, y), (x, y + d), (x + d, y)`, and the
+property of being corner-free the corners theorem and Roth's theorem.
```

Comment:
Is this missing a comma?

Reply:
Oh it should be
```suggestion
property of being corner-free.
```
The corners theorem and Roth's theorem will live in another file. See #9000.